api/src/services/reports-service.ts

```diff
@@ -22,4 +22,16 @@ export class ReportService extends DBService {
 
     return reportStatResponse[0];
   }
+
+  /**
+   * Get application report data.
+   *
+   * @return {*}  {Promise<IGetAppReport>}
+   * @memberof ReportService
+   */
+  async getAppReport(): Promise<IGetAppReport[]> {
+    const reportAppResponse = await Promise.all([this.reportRepository.getAppReportData()]);
+
+    return reportAppResponse[0];
```

Comment:
seems random to select the first here. Would be better suited to select the first in the SQL or just select the `row[0]` on return or error out if not found

Reply:
See the response from the query, it is a single query that returns an array of objects 
initially I had it as row[0] but it was not ion the response.